### DISCUSSION TITLE
修改 Mac 系统的二进制 plist 文件

### DISCUSSION THREAD

**Discussion by chen3feng** [chen3feng/article#95] [Show and tell]
Mac 系统采用 [plist](https://www.wikiwand.com/zh-hans/%E5%B1%9E%E6%80%A7%E5%88%97%E8%A1%A8) "属性列表"格式来保存一些配置信息，目前常见的有 XML 文本格式和二进制格式。

XML 格式的当然可以用任意文本编辑器查看和修改，但是二进制文件就不行了。

二进制 plist 可以用 xcode 来打开，还有一个收费的 PlistEdit Pro 软件。

VS Code 上有个扩展，叫 [Binary Plist](https://marketplace.visualstudio.com/items?itemName=dnicolson.binary-plist)，可以查看和编辑 plist 文件。

**Comment by kxn** [comment]
还可以用 propertree 来打开和编辑，虽然界面很丑陋，但是好在是python的，啥平台都能用

**Comment by chen3feng** [comment]
> 还可以用 propertree 来打开和编辑，虽然界面很丑陋，但是好在是python的，啥平台都能用

拜
